src/skin/options-layout.css

```diff
@@ -60,6 +44,13 @@ button
 
 .btn-silo div {
   padding-top: 10px;
```

Comment:
Spacing tweak?
```diff
 .btn-silo + .btn-silo {
   border-top: 1px solid #d3d3d3;
-  margin-top: 10px;
+  margin-top: 20px;
+  padding-top: 10px;
 }
 
 .btn-silo div {
-  padding-top: 10px;
   width: 350px;
   float: left;
 }
```

Reply:
I don't think we want to increase that top margin (that just puts more space between the buttons and the separator line below, and that spacing I think is fine), but moving the padding looks good.